### DISCUSSION TITLE
增量备份什么时候可以对已备份数据进行修改就太帅了

### DISCUSSION THREAD

**Discussion by LioDon** [ShunCai/QZoneExport#58] [Ideas]
没错，我还是那个空间数据量超过40G的QQ用户。还在用，每天都会在空间上更新。去年7月的备份数据就已经达到了43G，那时候的版本好像是1.0.8？转眼这都已经1.1.2了，看更新日志介绍说增加了说说照片九宫格等新功能。本想再做一次全量备份，可动辄40+G的数据量是直接劝退了……这要是直接能在老备份的基础上直接实现新功能和新数据的增量，那就真是太帅了。

**Comment by ShunCai** [comment]
1.2之前版本的增量备份我记得是支持跨版本的，只是1.2调整了数据存储所以没兼容老版本，而且，说真的，现在的增量备份功能有没有问题我都不清楚了，我一般都是全量备份的。

**Reply by LioDon** [reply]
数据量少怎么都好说，像我这40G起步的，全量备份简直是噩梦…光是第一步嗅探扫描就要好几个小时，而且还不一定就能抓取到所有的文件，最关键的是一丢就丢好几千个……下载就更累了…………我记得早年间腾讯官方有一个单独的上传照片的软件，那个软件好像还可以直接在线看云端的相册（好像下架好久了）。所以能不能在它的基础上拓展一下，不光能在线浏览还能做备份，不光能备份照片，还可以备份其他数据？……这是不是涉及到什么协议之类的东西没有权限啊………………

**Reply by ShunCai** [reply]
会丢？没试过丢数据的。
至于你说的其它数据指的是什么。

**Reply by ShunCai** [reply]
其实还好，全量备份的话，扔一个晚上让他自己备份就好。我6个g的，100m带宽的话，好像三个小时左右就完了，如果带宽更高的话就更快了。

**Reply by ShunCai** [reply]
至于下载文件的话，用Aria2是真的香。

**Reply by LioDon** [reply]
是真的会丢……一万六的照片在默认设置的数值下，我试了两次，只能扫出九千多张，剩下就不知道为什么扫不到。
之前用1.0.8的时候也有这样的问题，现在想了想，好像是相册单独备份，而且分两次，每次只选大概一半的量，才勉强完成。
印象中，就算是分两次全部完成的，还会出现几百张照片的文件格式丢失的情况，需要手动修改文件后缀。
至于 上边的说的其他数据，就是除了照片本身之外的东西，比如说文字类的说说、日志、评论、点赞之类的。说起来也挺好奇为什么那个相册软件会被官方遗弃，也不知道，如果真到了空间要关服务器的那天，腾讯是不是会出个官方的备份软件……
对了，能不能用类似网站复制工具那样的方式把空间给复制下来呢？那样是不是会最大化保留所有数据？你这个软件是不是可以往这个方向上尝试？

**Reply by ShunCai** [reply]
真的关服务器肯定会官方出工具的。

**Comment by LioDon** [comment]
对了，我注意到插件有错误信息提示。这错误信息怎么能导出来发你？如果对你有用的话……

**Reply by ShunCai** [reply]
截图就好
